content-warehouse/src/main/java/contentwarehouse/v1/QuickStart.java

```diff
@@ -0,0 +1,130 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package contentwarehouse.v1;
+
+// [START contentwarehouse_quickstart]
+
+
+import com.google.cloud.contentwarehouse.v1.CreateDocumentRequest;
+import com.google.cloud.contentwarehouse.v1.CreateDocumentResponse;
+import com.google.cloud.contentwarehouse.v1.CreateDocumentSchemaRequest;
+import com.google.cloud.contentwarehouse.v1.Document;
+import com.google.cloud.contentwarehouse.v1.DocumentSchema;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaServiceClient;
+import com.google.cloud.contentwarehouse.v1.DocumentSchemaServiceSettings;
+import com.google.cloud.contentwarehouse.v1.DocumentServiceClient;
+import com.google.cloud.contentwarehouse.v1.DocumentServiceSettings;
+import com.google.cloud.contentwarehouse.v1.LocationName;
+import com.google.cloud.contentwarehouse.v1.Property;
+import com.google.cloud.contentwarehouse.v1.PropertyDefinition;
+import com.google.cloud.contentwarehouse.v1.RequestMetadata;
+import com.google.cloud.contentwarehouse.v1.TextArray;
+import com.google.cloud.contentwarehouse.v1.TextTypeOptions;
+import com.google.cloud.contentwarehouse.v1.UserInfo;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class QuickStart {
+  
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectNumber = "your-project-number";
+    String location = "your-region"; // Format is "us" or "eu".
+    String userId = "your-user-id"; // Format is user:<user-id>
+    quickStart(projectNumber, location, userId);
+  }
+
+  public static void quickStart(String projectId, String location, String userId)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+
+    String endpoint = String.format("%s-contentwarehouse.googleapis.com:443", location);
+    DocumentSchemaServiceSettings documentSchemaServiceSettings = 
+         DocumentSchemaServiceSettings.newBuilder().setEndpoint(endpoint).build(); 
+  
+    // Create a Schema Service client
+    try (DocumentSchemaServiceClient documentSchemaServiceClient =
+        DocumentSchemaServiceClient.create(documentSchemaServiceSettings)) {
+      /*  The full resource name of the location, e.g.:
+      projects/{project_number}/locations/{location} */
+      String parent = LocationName.format(projectId, location);
+
+      // Create Document Schema with Text Type Property
```

Comment:
nit: I would provide a link to documentation that describes what the "Text Type Property" is.

Reply:
I added a link to this documentation for managing document schemas which goes into the different property types. 
https://cloud.google.com/document-warehouse/docs/manage-document-schemas